tests/test_Account.py

```diff
@@ -102,3 +102,20 @@ async def test_public_key_setter(account_factory):
 
     execution_info = await account.get_public_key().call()
     assert execution_info.result == (other.public_key,)
+
+
+@pytest.mark.asyncio
+async def test_is_account(account_factory):
+    starknet, account = account_factory
+
+    # account contract
+    execution_info = await account.is_account().call()
+    assert execution_info.result == (1,)
+
+    other = await starknet.deploy(
+        "contracts/token/utils/ERC721_Holder.cairo"
+    )
+
+    # non-account contract
+    execution_info = await other.is_account().call()
+    assert execution_info.result == (0,)
```

Comment:
I don't think it makes sense to test `is_account` on a random contract since it provides no info/guarantees about the Account contract (the focus of this test suite). With [my previous comment](https://github.com/OpenZeppelin/cairo-contracts/pull/133#pullrequestreview-853110840) I meant testing `safeTransferFrom` when the receiver is not an account.

Reply:
Ah my mistake. I believe our `safeTransferFrom` tests already test receiver contracts [here](https://github.com/andrew-fleming/cairo-contracts/blob/add-is-account/tests/test_ERC721_Mintable.py#L602-L609) when it's not an account. We also have a test for an account that does not expose `is_account` [here](https://github.com/andrew-fleming/cairo-contracts/blob/add-is-account/tests/test_ERC721_Mintable.py#L736). I suppose another test we can add is for contracts that expose `is_account` but do not support IERC721Receiver interface